openedx/core/djangoapps/plugins/plugin_slots.py

```diff
@@ -0,0 +1,106 @@
+from importlib import import_module
+from logging import getLogger
+
+from importlib_metadata import ModuleNotFoundError
+
+from openedx.core.lib.cache_utils import process_cached
+from . import constants, registry
+from .plugin_contexts import get_plugins_view_context
+
+log = getLogger(__name__)
+
+COMMON_WHITELIST = ['request', 'current_url']
+
+
+def get_content_for_slot(project_type, slot_name, raw_context):
+    """
+    Returns a list of additional content for a view slot. Will check if any plugin apps
+    have that view in their view_context_config, and if so will call their
+    selected function to get their context dicts.
+    Params:
+        project_type: a string that determines which project (lms or studio) the view is being called in. See the
+            ProjectType enum in plugins/constants.py for valid options
+        slot_name: a string that determines which slot the plugin will render content for. These are unique for
+            each project type.
+    """
+    aggregate_slot_content = u""
+    slot_functions = _get_cached_slot_functions_for_view(project_type, slot_name)
+
+    # Each view can pass the whitelisted part of the context, in the context itself.
+    context_whitelist = raw_context.get('context_whitelist', [])
```

Comment:
I'd like to propose calling this `context_allow_list` instead of `context_whitelist` - whitelist has problematic connotations. (Same with `COMMON_WHITELIST` above)

Reply:
Certainly, thanks for pointing it out.